packages/icons/src/paths/ChevronsLeft.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import createSvgIcon from '../utils/createSvgIcon';
+
+export const ChevronsLeft = createSvgIcon(
+  <React.Fragment>
+    <path
+      clip-rule="evenodd"
+      d="M6.41427 12.0711L12.7782 5.70718L11.364 4.29297L3.58584 12.0711L11.364 19.8493L12.7782 18.4351L6.41427 12.0711Z"
+      fill="#303030"
+      fill-rule="evenodd"
```

Comment:
Hey, can you check, this might return console errors, possibly would need a change from fill-rule to fillRule and also same for clip-rule to clipRule in all svgs components you've created